packages/utils/src/RBAC.js

```diff
@@ -0,0 +1,93 @@
+
+export async function getRBAC(applicationName = '') {
+    const insights = window.insights;
+    if (
+        insights &&
+    insights.chrome &&
+    insights.chrome.auth &&
+    insights.chrome.auth.getUser &&
+    insights.chrome.getUserPermissions
+    ) {
+        const user = await insights.chrome.auth.getUser();
+        const permissions = await insights.chrome.getUserPermissions(applicationName);
+        return {
+            isOrgAdmin: user.identity.user.is_org_admin,
+            permissions
+        };
+    }
+
+    return { isOrgAdmin: false, permissions: null };
+}
+
+export function doesHavePermissions(RBACResults, permissionList) {
+    const { isOrgAdmin, permissions } = RBACResults;
+    if (isOrgAdmin) {
+        return true;
+    }
+
+    if ([ null, undefined ].includes(permissions)) {
+        return false;
+    }
+
+    return permissions.some(function(access) {
+        return permissionList.includes(access.permission);
+    });
+}
+
+export const RBAC_ACTION_TYPES = { PENDING: 'RBAC_PENDING', SUCCESS: 'RBAC_SUCCESS', FAILURE: 'RBAC_FAILURE' };
```

Comment:
Can you move the redux stuff to specific file? This way we won't import them if not needed. Also I would prefer hooks in here rather than redux.

```JS
import { useState, useEffect } from 'react';
import { getRBAC, doesHavePermissions } from './RBAC';

export function usePermissions(appName, permissionsList) {
  const [permissions, setPermissions] = useState({});
  useEffect(() => {
    (async () => {
        const { isOrgAdmin, permissions: userPermissions } = await getRBAC(appName);
        setPermissions({
           isOrgAdmin,
           permissions: userPermissions,
           hasAccess: doesHavePermissions(userPermissions, permissionsList)
        });
    })()
  }, [ appName ])
  return permissions;
}
```
And usage would be
```JSX
import React from 'react'
import { usePermissions } from '@redhat-cloud-services/frontend-components-utilities/files/permissions';

export const () => {
  const { hasAccess } = usePermissions('inventory', [ 'inventory:detail:read' ]);
  return <div>{hasAccess && <span>User has access</span>}</div>;
}
```